.goreleaser.yml

```
@@ -72,6 +72,8 @@ brews:
       bin.install "exo"
       man1.install Dir["manpage/exo*.1"]
       bash_completion.install "contrib/completion/bash/exo"
+      fish_completion.install "contrib/completion/fish/exo"
+      zsh_completion.install "contrib/completion/zsh/exo"
```

```suggestion
      zsh_completion.install "contrib/completion/zsh/_exo"
```

for me adding the file as exo does not work, but installing it manually as _exo works. I do not know if the goreleaser does add the underscore implicitly while installing